modules/core/src/core/stats-manager.js

```diff
@@ -0,0 +1,17 @@
+import {Stats} from 'probe.gl';
+
+class StatsManager {
```

Comment:
There is a global object `luma.stats` that tracks the count of resource objects by type. Worth consolidating?

Reply:
Definitely! Hadn't notice that.